### PR TITLE
Readme: Update IntelliJ import instructions

### PR DESCRIPTION
**Pull request by pepper-jk**
Since writing the Development section in the README, the IntelliJ UI for importing a project has changed, especially in the welcome dialog, which is what new IntelliJ users see first.
This MR aims to address these changes, in order to make it easier for new developers to contribute to ORT, and also attempt to make the instructions more legible.

Signed-off-by: Jens Keim <jens.keim@forvia.com>

### CODE REVIEW

**Comment by pepper-jk** [comment]
Preview of the new Readme content: https://github.com/pepper-jk/ort/tree/import_instructions#development

File: README.md [hunk 1]
```diff
@@ -727,13 +727,13 @@ For IDE development we recommend the [IntelliJ IDEA Community Edition](https://w
 can directly import the Gradle build files. After cloning the project's source code recursively, simply run IDEA and use
 the following steps to import the project.
 
-1. From the wizard dialog: Select *Import Project*.
+1. From the welcome dialog: Select `Open`.
```
Comment (sschuberth) [inline README.md:730]: We should probably say "*Welcome* dialog" here for consistency with "*Open Project* dialog" below.

File: README.md [hunk 2]
```diff
@@ -727,13 +727,13 @@ For IDE development we recommend the [IntelliJ IDEA Community Edition](https://w
...
 
-   From a running IDEA instance: Select *File* -> *New* -> *Project from Existing Sources...*
+   From a running IDEA instance: Select `File`>`New`>`Project from Existing Sources...`
```
Comment (sschuberth) [inline README.md:732]: If not keeping "->", can we at least keep the spaces around ">" for readability of the raw text?

**Review by sschuberth — CHANGES_REQUESTED** [review]

**Comment by sschuberth** [comment]
Also, please follow Conventional Commits for both commit message titles. I propose to use a prefix of "docs(README): ...".

File: README.md (hunk 2 above)
Comment (pepper-jk) [reply README.md]: Changes were applied.

File: README.md (hunk 1 above)
Comment (pepper-jk) [reply README.md]: Change was applied.

**Comment by sschuberth** [comment]
> Preview of the new Readme content: https://github.com/pepper-jk/ort/tree/import_instructions#development

Thanks. FYI, that can also be seen from the [rich diff](https://github.com/oss-review-toolkit/ort/pull/7044/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5).

**Comment by sschuberth** [comment]
> > Also, please follow Conventional Commits for both commit message titles. I propose to use a prefix of "docs(README): ...".
> 
> I applied the prefix you suggested.

You did not quite 😉 "doc*s*" really needs to be plural...

**Comment by pepper-jk** [comment]
> > > Also, please follow Conventional Commits for both commit message titles. I propose to use a prefix of "docs(README): ...".
> > 
> > 
> > I applied the prefix you suggested.
> 
> You did not quite wink "doc_s_" really needs to be plural...

My bad. :see_no_evil: Fixed now. :crossed_fingers:

**Review by sschuberth — APPROVED** [review]

**Pull request merged**